### DISCUSSION TITLE
Restart command

### DISCUSSION THREAD

**Discussion by arboeh** [goecharger/go-eCharger-API-v2#49] [Q&A] [answered]
I'm searching for the restart-command for go-e for API V2.
For V1 I can use http://xxx.xxx.xxx.xxx/mqtt?payload=rst=1 and it works from local network with Home Assistant...

**Comment by syssi** [comment, marked as answer]
The "rst" key is supported at API v2 (firmware `0.53.1`) too: https://github.com/syssi/homeassistant-goecharger-mqtt/blob/main/custom_components/goecharger_mqtt/definitions/button.py#L26-L35
